src/test/scala/com/mozilla/telemetry/streaming/TestAggregator.scala

```diff
@@ -100,4 +100,55 @@ class TestAggregator extends FlatSpec with Matchers{
     results("window_start").asInstanceOf[Timestamp].getTime should be <= (TestUtils.testTimestampMillis)
     results("window_end").asInstanceOf[Timestamp].getTime should be >= (TestUtils.testTimestampMillis)
   }
-}
+
+  "The aggregator" should "handle new style experiments" in {
+    val spark = SparkSession.builder()
```

Comment:
Instead of creating spark here, move it to a `beforeAll()` function. Remove it from the other test as well. Add an `afterAll()` function which calls `spark.stop()`. For an example of this, see [spark-hyperloglog tests](https://github.com/mozilla/spark-hyperloglog/blob/master/src/test/scala/com/mozilla/spark/sql/hyperloglog/test/HyperLogLog.scala). Note you need to mixin the `BeforeAndAfterAll` trait.